src/main/java/teammates/logic/backdoor/BackDoorServlet.java

```diff
@@ -261,8 +262,11 @@ private String executeBackendAction(HttpServletRequest req, String action)
             return backDoorLogic.getFeedbackQuestionAsJson(feedbackSessionName, courseId, qnNumber);            
         }  else if (action.equals(OPERATION_GET_FEEDBACK_QUESTION_FOR_ID_AS_JSON)) { 
             String questionId = req.getParameter(PARAMETER_FEEDBACK_QUESTION_ID);
-            return backDoorLogic.getFeedbackQuestionForIdAsJson(questionId);            
-        }  else if (action.equals(OPERATION_GET_FEEDBACK_RESPONSE_AS_JSON)) { 
+            return backDoorLogic.getFeedbackQuestionForIdAsJson(questionId);
+        } else if (action.equals(OPERATION_DELETE_FEEDBACK_QUESTION)) {
```

Comment:
You will have to add a test case for this in BackDoorTest, possibly in a separate issue.